library/Rules/DateTimeDiff.php

```diff
@@ -35,7 +35,11 @@ final class DateTimeDiff extends Standard
 
     private readonly Validatable $rule;
 
-    /** @param string $type "years"|"months"|"days"|"hours"|"minutes"|"seconds"|"microseconds" */
+    /** 
+     * @param string $type "years"|"months"|"days"|"hours"|"minutes"|"seconds"|"microseconds" 
+     * @param string|null $format Example: "Y-m-d H:i:s.u"
+     * @param DateTimeImmutable|null $now The value that will be compared to the input
```

Comment:
I'm a against adding documentation to the code like that, I think the name of the parameter should be pretty obvious for what it does, otherwise we should do that for everything. I understand that it can be useful to add some documentation here and there, but for this specific method, I would rather not have it.